admin/class-admin.php

```diff
@@ -51,16 +51,6 @@ public function __construct() {
 			add_filter( 'wpseo_accessible_post_types', array( 'WPSEO_Post_Type', 'filter_attachment_post_type' ) );
 		}
 
-		$this->admin_features = array(
-			// Google Search Console.
-			'google_search_console' => new WPSEO_GSC(),
-			'dashboard_widget'      => new Yoast_Dashboard_Widget(),
-		);
-
-		if ( WPSEO_Metabox::is_post_overview( $pagenow ) || WPSEO_Metabox::is_post_edit( $pagenow ) ) {
```

Comment:
I would move this check to the `WPSEO_Primary_Term_Admin`. I saw you already do this in this pull. And keep these code at this location. It doesn't hurt and won't give us the need to do an `end` call of the array.